src/PrestaShopBundle/Form/Admin/Type/DateRangeType.php

```diff
@@ -38,28 +39,30 @@ class DateRangeType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('from', DatePickerType::class, [
-                'required' => false,
-                'attr' => [
-                    'placeholder' => 'From',
-                ],
-                'translation_domain' => 'Admin.Global',
-                'date_format' => $options['date_format'],
-            ])
+            ->add('from', DatePickerType::class, $this->getFromFieldOptions($options))
```

Comment:
Hmmm, I get it now I didn't realize we had a generic form type to handle date ranges It makes sense now why you created a generic component. It's a good call, my bad!

But we just need to find a way to make it more generic and avoid using specific ids like `#specific_price_date_range_unlimited` Instead, we should use a selector based on a CSS class maybe.